File: backend/msreserva/src/main/java/com/dac/msreserva/model/ReservaConsultaId.java
package com.dac.msreserva.model;

import jakarta.persistence.Embeddable;
import java.io.Serializable;

@Embeddable
public class ReservaConsultaId implements Serializable {

    private String codigo;
    private String codigo_voo;

    // Getters and Setters, equals, and hashCode
}

File: backend/reserva-service/src/main/kotlin/br/ufpr/dac/reserva_service/domain/embeddable/ReservaConsultaId.kt
package br.ufpr.dac.reserva_service.domain.embeddable

import jakarta.persistence.Embeddable
import java.io.Serializable

@Embeddable
data class ReservaConsultaId(
    val codigo: String,
    val codigo_voo: String
) : Serializable
